pkg/orchestration/wiring/testdata/asapkey.basicdep.bundle.yaml

```diff
@@ -68,9 +68,16 @@ spec:
           computeType: kubeCompute
           oapResourceName: kubecompute-simple-iamrole
           serviceEnvironment:
-            lowPriorityPagerdutyEndpoint: https://events.pagerduty.com/adapter/cloudwatch_sns/v1/12312312312312312312312312312312
+            alarmEndpoints:
```

Comment:
actually it's not used in iam role template, so we could just not send them maybe?